### PR TITLE
FAQ Model

### PR DESCRIPTION
**Pull request by jovanbodroza**
Created jobs app inside Django project and added model for FAQ

### CODE REVIEW

File: opportunity_job_api/src/apps/jobs/apps.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class JobsConfig(AppConfig):
+    default_auto_field = 'django.db.models.BigAutoField'
+    name = 'jobs'
```
Comment (roknicmilos) [inline opportunity_job_api/src/apps/jobs/apps.py:6]: Use double quotes

File: opportunity_job_api/src/apps/jobs/models.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+from django.db import models
+from apps.common.models import BaseModel
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+# Create your models here.
```
Comment (roknicmilos) [inline opportunity_job_api/src/apps/jobs/models.py:5]: You can remove this comment

File: opportunity_job_api/src/apps/jobs/models.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        default=False
+    )
+    display_to_employers = models.BooleanField(
+        verbose_name=_("display to employers"),
+        default=False
+    )
+    display_to_applicants = models.BooleanField(
+        verbose_name=_("display to applicants"),
+        default=False
+    )
```
Comment (roknicmilos) [inline opportunity_job_api/src/apps/jobs/models.py:32]: Add an empty line (applies to all files)

File: opportunity_job_api/src/apps/jobs/tests.py [hunk 4]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```
Comment (roknicmilos) [inline opportunity_job_api/src/apps/jobs/tests.py:3]: You can remove this file

**Review by roknicmilos — CHANGES_REQUESTED** [review]

File: opportunity_job_api/src/apps/jobs/apps.py (hunk 1 above)
Comment (jovanbodroza) [reply opportunity_job_api/src/apps/jobs/apps.py]: Corrected

File: opportunity_job_api/src/apps/jobs/models.py (hunk 2 above)
Comment (jovanbodroza) [reply opportunity_job_api/src/apps/jobs/models.py]: Removed

File: opportunity_job_api/src/apps/jobs/models.py (hunk 3 above)
Comment (jovanbodroza) [reply opportunity_job_api/src/apps/jobs/models.py]: Empty lines added

File: opportunity_job_api/src/apps/jobs/tests.py (hunk 4 above)
Comment (jovanbodroza) [reply opportunity_job_api/src/apps/jobs/tests.py]: File tests.py removed

File: opportunity_job_api/src/apps/jobs/models.py [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+        verbose_name = _("FAQ")
+        verbose_name_plural = _("Frequently Asked Questions")
+
+    question = models.CharField(
+        verbose_name=_("question"),
+        max_length=250,
+    )
+    answer = models.CharField(
+        verbose_name=_("answer"),
+        max_length=250,
```
Comment (roknicmilos) [inline opportunity_job_api/src/apps/jobs/models.py:18]: Anser neka bude `TextField` za to sto ce neki odgovori biti dosta veliki

File: opportunity_job_api/src/apps/jobs/models.py [hunk 6]
```diff
@@ -0,0 +1,31 @@
+from django.db import models
+from apps.common.models import BaseModel
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+class FAQ(BaseModel):
+
```
Comment (roknicmilos) [inline opportunity_job_api/src/apps/jobs/models.py:7]: Dodaj [django-modeltranslations](https://django-modeltranslation.readthedocs.io/en/latest/) u projekat i registruj ovaj model kao "translatabilan", tj. dodaj `question` i `answer` fields kao "translatabilne".

Primer:
image

**Review by roknicmilos — CHANGES_REQUESTED** [review]

**Pull request closed without merging**